### ISSUE TITLE
Issue when using ResponseDetective with Carthage

### ISSUE THREAD

**Issue by adrianbukros** [netguru/ResponseDetective#63]
Hello! We are having this issue since yesterday when trying to use the project through Carthage:

```
A shell task (/usr/bin/env git clone --bare --quiet https://github.com/netguru/xcconfigs.git /Users/adrian/Library/Caches/org.carthage.CarthageKit/dependencies/xcconfigs) failed with exit code 128:
fatal: could not read Username for 'https://github.com': terminal prompts disabled
```

Any chance that the access rights for the repo https://github.com/netguru/xcconfigs.git changed?

**Comment by akashivskyy** [comment]
Yes, it seems that xcconfigs repo has been made private. I don't work at Netguru anymore, and I haven't been given prior notice of this change. Fortunately, I have a local clone and the repository was MIT-licensed. I'll just include the config files directly in ResponseDetective instead of pulling them from that repository. Hold on.

**Comment by akashivskyy** [comment]
Side note: I'm not sure how much longer I will be able to maintain ResponseDetective. I don't get paid for this anymore and I'm effectively maintaining a project for free under my ex-employer's brand (this is strange). I'm not sure what will happen once I resign from being a maintainer, there's a high chance this will become abandonware or just disappear randomly as xcconfigs did. I suggest migrating off ResponseDetective to other tools, such as [FLEX](https://github.com/FLEXTool/FLEX), or keeping a fork just in case.

**Comment by akashivskyy** [comment]
[Released in version 1.5.0](https://github.com/netguru/ResponseDetective/releases/tag/1.5.0). Your project should now build. Thank you for bringing this up.

**Comment by adrianbukros** [comment]
Thank you for solving this quickly!
